### ISSUE TITLE
Bot Discussion: What Should Penny Be Able To Respond To?

### ISSUE THREAD

**Issue by JelaniThompson** [the-heap/penny#23]
Hey folks! 👋

I was able to finish up getting [Penny to respond in DMs](https://github.com/the-heap/penny/pull/22) but right now we're kind of limited in terms of what she replies to. We have `give prompt` and `submit prompt` in addition to the default response.

I had a few potential silly responses we could have but for now we're probably only looking for ones that would expand on the core functionality. 

Anyone have any ideas?

Reference: Last checkbox on #5

**Comment by teesloane** [comment]
👋 👋 👋 👋

Thanks for making this issue! 

Yes, so right now we don't have too many responses that Penny can make.

Penny isn't actually supposed to respond to much else that what it does already. there is one remaining big block:

- when people actually draw something (ie. the prompt), they will take a picture of it
- A user should be able to DM penny an upload of their drawing. 
- Then we'll handle having penny grab that photo and compile what other photos may exist within a time frame into a gallery. Does that make sense?

**Comment by JelaniThompson** [comment]
Yep, makes sense! Creating that gallery will definitely be pretty fun 👌
